shared/util/zustand.tsx

```diff
@@ -29,6 +30,18 @@ export const createZustand = <T extends HasReset>(
   const store = create<T, [['zustand/immer', never]]>(f)
   // includes dispatch, custom overrides typically don't
   const initialState = store.getState()
+  // wrap so we log all exceptions
+
+  const dispatches = Object.keys(initialState.dispatch)
+  const unsafeISD = (initialState as any).dispatch as any
+  for (const d of dispatches) {
+    const orig = unsafeISD[d] as any
+    if (typeof orig === 'function') {
+      unsafeISD[d] = wrapErrors(orig, d)
+      // copy over things like .cancel etc
+      Object.assign(unsafeISD[d], orig)
```

Comment:
tricky: debounced etc also make `.cancel` methods on the functions themselves so ensure we get all the extra stuff